modules/virt-deleting-virt-cli.adoc

```diff
@@ -16,7 +16,7 @@ You can delete {VirtProductName} by using the CLI.
 
 [NOTE]
 ====
-When you delete the subscription of the {VirtProductName} operator in the OLM by using the CLI, the ClusterServiceVersion (CSV) object is not deleted from the cluster. To completely uninstall {VirtProductName}, you must explicitly delete the CSV.
+When you delete the subscription of the {VirtProductName} operator in the OLM by using the CLI, the cluster service version (CSV) object is not deleted from the cluster. To completely uninstall {VirtProductName}, you must explicitly delete the CSV.
```

Comment:
This says "object," so I think it would be `ClusterServiceVersion`. Another edge case to look for: parentheticals interrupting the phrase